snap/snapcraft.yaml

```diff
@@ -54,9 +54,15 @@ parts:
         snapcraftctl build
         TRIPLET_PATH="$SNAPCRAFT_PART_INSTALL/usr/lib/$(gcc -print-multiarch)"
         LIBSODIUM=$(readlink -n $TRIPLET_PATH/libsodium.so.18)
+        # Remove so the link can be recreated on re-builds
+        [ -f "$TRIPLET_PATH/libsodium.so" ] && rm "$TRIPLET_PATH/libsodium.so"
         ln -s $LIBSODIUM $TRIPLET_PATH/libsodium.so
-        patch -d $SNAPCRAFT_PART_INSTALL/lib/python3.5/site-packages -p1 < $SNAPCRAFT_STAGE/pyyaml-support-high-codepoints.diff
-        patch $SNAPCRAFT_PART_INSTALL/usr/lib/python3.5/ctypes/__init__.py $SNAPCRAFT_STAGE/ctypes_init.diff
+        # These checks are ugly but allow us to rebuild.
+        [ -f $SNAPCRAFT_PART_INSTALL/lib/python3.5/site-packages/yaml/emitter.py.orig ] && mv $SNAPCRAFT_PART_INSTALL/lib/python3.5/site-packages/yaml/emitter.py.orig $SNAPCRAFT_PART_INSTALL/lib/python3.5/site-packages/yaml/emitter.py
```

Comment:
I see no `prime` filter to remove these, nor do I see logic in `override-prime` to remove them-- probably best not to ship .orig files in the snap, no?

Reply:
I thought it would be rather harmless, but sure.